paperpress.js

```diff
@@ -114,13 +115,33 @@ Paperpress.prototype._fileToItem = function (file) {
 
 	var item = {
 		title: name,
-		slug: slug
+		slug: slug,
+		content: fileContent,
+		path: null
 	}
 
-	item.path = this.pathBuilder(item, file.collectionName)
-	item.content = (fileType === '.md') ? marked.render(fileContent) : fileContent
+	if (frontMatter.test(fileContent)) {
+		var contentData = frontMatter(fileContent)
+		var attr = contentData.attributes
+
+		item = {
```

Comment:
What do you think about adding a util function to extend objects? This will also cover the concern about fallback values for the `item` object.

```
Paperpress.prototype._objectExtend: function(destination, source) {
  for (var property in source) {
    if (source.hasOwnProperty(property)) {
      destination[property] = source[property]
    }
  }
  return destination
}
```